.github/workflows/ci.yaml

```diff
@@ -14,6 +14,8 @@ jobs:
     strategy:
       matrix:
         php-version:
+          - 7.2
```

Comment:
I will remove PHP 7.2 support later again, because its support ends very soon.

Reply:
Already discussed that with Florian.

Take a look at Ubuntu for example. Just like TYPO3 ELTS, support isn't over because PHP Core does no longer provide Support. There are many systems that still provide support and are in use out there.
I would not drop the support unless there is a good reason.


Ubuntu 18.04 LTS has support until 2028-04 which has PHP 7.2 https://packages.ubuntu.com/bionic/php/php.

Reply:
With the same argument you still need to support IE 11, because Microsoft has some ultra long term support for it (at least partially). Feel free to use PHP 5.6 with Debian as well.
Old is old, full stop. If you rely on unmaintained stuff, feel free, its your own risk. The additional support of outdated stuff by those distros is nothing else than a heart-lung machine. (and a huge PITA)

I will NOT take the time to fix anything in this regard, once PHP drops support for a version.
That's why I will kick the tests on this version. Anybody is free to use stuff with an old version.